### DISCUSSION TITLE
Share button is only visible on laptop website but not on mobile web. Any way to enable it for mobile?

### DISCUSSION THREAD

**Discussion by aapwikip** [requarks/wiki#6629] [Help / Questions]
Share button is only visible on laptop website but not on mobile web. 

Any way to enable it for mobile web?
And also change the location to top of the page? 
Having Twitter/Facebook/Whatapps etc icon appear individually would be icing on the cake

![image](https://github.com/requarks/wiki/assets/134777760/30ca37d2-0cd5-4906-b98a-4fe84db2cad4)

**Comment by NGPixel** [comment]
That's not possible no. This is redundant on mobile as your mobile browser already have a share button... so it wouldn't make sense to waste the very limited space on mobile to display a duplicate button.

**Reply by aapwikip** [reply]
https://github.com/requarks/wiki/discussions/6629#discussioncomment-6690418

**Comment by aapwikip** [comment]
Thanks for replying. While I agree that share button is already available on mobile but it is a multi click access which makes it unusable for most which is happening with 99.9% of the user base, at-least for us.

So we intend to enable a one click share button, possibly direct links to FB, Twitter, WA, LinkedIn etc and share button for rest

Since Wiki.js platform already supports this feature, please suggest a way to enable this either via code/css injection from admin settings or minor changes in code.

**Comment by aapwikip** [comment]
Most mobile version of websites have share buttons too including nytimes.com

Good thing is that Wiki.js platform already supports this feature
WE just need to figure out a way to enable this either via code/css injection from admin settings or custom changes in code.

Please suggest a way out

**Reply by NGPixel** [reply]
Like I said, this is not possible at the moment. It's not something that can changed via CSS/JS injection alone.

**Reply by aapwikip** [reply]
Requesting & hoping to have this flexibility in v3.x

**Comment by aapwikip** [comment]
Requesting & hoping to have this functionality & flexibility in v3.x
